Emby.Server.Implementations/Localization/iso6392.txt

```diff
@@ -349,7 +349,8 @@ pli||pi|Pali|pali
 pol||pl|Polish|polonais
 pon|||Pohnpeian|pohnpei
 por||pt|Portuguese|portugais
-pob||pt-br|Portuguese (Brazil)|portugais
+pop||pt-pt|Portuguese (Portugal)|portugais (pt-pt)
```

Comment:
Where did you get pop? When I look it up I get this https://iso639-3.sil.org/code/pop

Reply:
Like many others on that file, pob is a made up code (https://iso639-3.sil.org/code/pob).

 I just followed suit for a quick fix before #6750 can get done.

Reply:
ISO 639 really shouldn't be used for this, instead I believe we should be mirroring [Weblate country data](https://github.com/WeblateOrg/language-data), but that is a different issue.

Reply:
I don't know. But you are editing a file called `iso6392.txt` (aka. ISO639-2), so I don't feel great about adding a bogus three letter code.

Reply:
Half of the file is bogus (take a look at French Canadian and Brazilian Portuguese per example). This standard isn't made for localisation and somewhere along the line instead of fixing the implementation it simply got these additions.


This is meant as a way to enable European Portuguese to be used, not a permanent fix as I've mentioned in the issue.

https://www.loc.gov/standards/iso639-2/php/code_list.php here is the unadulterated source.

Reply:
> This standard isn't made for localisation and somewhere along the line instead of fixing the implementation it simply got these additions.

What do you propose?

Reply:
In the eventual PR addressing #6750, to replace this file entirely with this https://github.com/WeblateOrg/language-data/blob/main/languages.csv. This also contains the info needed for handling of plural forms in localization, which is an issue with Jellyfin as mentioned [here](https://github.com/jellyfin/jellyfin-vue/issues/1354).

As for this PR, it's just a quick fix so I can get my instance loading the correct translation.